src/components/Discussion/Tree/CommentNode.js

```diff
@@ -0,0 +1,529 @@
+import React, { useMemo } from 'react'
```

Comment:
It's a bit hard to know what has truly changed in the code because of the renaming... any clues as to what I need to focus on?

Reply:
Two things were changed: The ExpandIcons were changed and the callout menu was added (Be sure to checkout the menu variable and the useMemo hook).